src/test/java/org/selenium/pages/tests/BaseTest.java

```diff
@@ -0,0 +1,48 @@
+package org.selenium.pages.tests;
+
+import org.junit.Before;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.selenium.driver.Driver;
+import org.selenium.pages.*;
+import org.selenium.utils.JsonParser;
+
+public abstract class BaseTest {
+    protected static WebDriver driver;
+    protected static WebDriverWait wait;
+    protected static JsonParser parser = new JsonParser();;
+    protected static MainPage mainPage;
+    protected static MyAccountPage account;
+    protected static RegistrationPage registration;
+    protected static LogInPage signIn;
+    protected static DressesPage dresses;
+    protected static CartPage cart;
+    protected static WishListPage wishList;
+    protected static ProductDetailPage product;
```

Comment:
Also, please name the variables in the same way as the name of the class. It will improve code readability